src/Security/Authorization/Core/src/Policy/AuthorizationEndpointConventionBuilderExtensions.cs

```diff
@@ -13,38 +13,25 @@ namespace Microsoft.AspNetCore.Builder
     public static class AuthorizationEndpointConventionBuilderExtensions
     {
         /// <summary>
-        /// Adds authorization policies with the specified <see cref="IAuthorizeData"/> to the endpoint(s).
+        /// Adds the default authorization policy to the endpoint(s).
```

Comment:
@JamesNK - I reordered this.

I also made a different (and I think more inutitive decision) than what we discussed. Open to discuss further. 

Check out the docs on the other methods here.

Reply:
This will use the default policy if you don't specify any. So basically:

```
builder.RequireAuthorization() == builder.RequireAuthorization(Array.Empty<string>())
```

This made more sense to me then trying to force you to pass in a policy/data.

Reply:
Also will add a test for this behaviour since there aren't any.